dashboard/src/org/labkey/dashboard/ProjectAdminsTable.java

```diff
@@ -0,0 +1,274 @@
+            Container container = ContainerManager.getForId(entityId);
+
+            if (container != null)
+            {
+                List<UserPrincipal> allAdmins = getAdmins(container);
+
+                StringBuilder display = new StringBuilder();
+
+                String space = "";
+
+                if (allAdmins.size() > 0)
+                {
+                    for(UserPrincipal user: allAdmins)
+                    {
+                        display.append(space);
+                        display.append(getUserDetails(container, user));
+                        space = ",&nbsp;";
+
+                    }
+
+                    out.write(display.toString());
+                    return;
+                }
+            }
+            super.renderGridCellContents(ctx, out);
+        }
+
+        @NotNull
+        private List<UserPrincipal> getAdmins(Container container)
+        {
+            List<UserPrincipal> allAdmins = new ArrayList<>();
+
+            SecurityPolicy policy = SecurityPolicyManager.getPolicy(container);
+            Role projectAdminRole = RoleManager.getRole(ProjectAdminRole.class);
+            Role siteAdminRole = RoleManager.getRole(SiteAdminRole.class);
+            Map<String, Map<PrincipalType, List<UserPrincipal>>> roleMap = policy.getAssignmentsAsMap();
+
+            for (String roleName : roleMap.keySet())
+            {
+                Role role = RoleManager.getRole(roleName);
+                if (role == null || (!(role.equals(projectAdminRole) || role.equals(siteAdminRole))))
+                {
+                    continue;
+                }
+                Map<PrincipalType, List<UserPrincipal>> projAdmins = roleMap.get(roleName);
+                for (PrincipalType pType : projAdmins.keySet())
+                {
+                    if (pType.equals(PrincipalType.USER))
+                    {
+                        allAdmins.addAll(projAdmins.get(pType));
+                    }
+                    else if (pType.equals(PrincipalType.GROUP))
+                    {
+                        List<UserPrincipal> adminGroups = projAdmins.get(pType);
+                        for(UserPrincipal group: adminGroups)
+                        {
+                            Group grp = SecurityManager.getGroup(group.getUserId());
+                            Set<User> grpMembers = SecurityManager.getAllGroupMembers(grp, MemberType.ACTIVE_USERS);
+                            allAdmins.addAll(grpMembers);
+                        }
+                    }
+                }
+            }
+            return allAdmins;
+        }
+
+        private String getUserDetails(Container container, UserPrincipal member)
+        {
+            ActionURL url = PageFlowUtil.urlProvider(UserUrls.class).getUserDetailsURL(container, member.getUserId(), null);
+            return new StringBuilder().append("<a href='").append(url.getLocalURIString()).append("'>").append(member.getName()).append("</a>").toString();
```

Comment:
Need to HTML-encode the name. Consider using our new-ish org.labkey.api.util.DOM builder utility